app/containers/StudyPage/StudyPage.tsx

```diff
@@ -443,7 +447,20 @@ class StudyPage extends React.Component<StudyPageProps, StudyPageState> {
   };
 
   renderReviewsSummary = (data: StudyPageQuery | undefined) => {
-    if (!data || !data.study) return null;
+    if (!data || !data.study) {
+      return <ReviewsWrapper>
+        <div>
+          <ReactStars
+            count={5}
+            color2={starColor}
+            edit={false}
+            value={0}
+          />
+          <div>{'0 Reviews'}</div>
+        </div>
+      </ReviewsWrapper>;
+    }
+
```

Comment:
To avoid the reviews on the sidebar being jumpy, it just creates a dummy object. This might be a tad inefficient.